apps/extension/webpack.config.js

```diff
@@ -0,0 +1,146 @@
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import webpack from 'webpack';
+
+const PRAX = 'lkpmkhpnhknhmibgnmmhdhgdilepfghe';
+
+const IDB_VERSION = 23;
+const USDC_ASSET_ID = 'reum7wQmk/owgvGMWMZn/6RFPV24zIKq3W6In/WwZgg=';
+
+const MINIFRONT_URL = 'https://app.testnet.penumbra.zone/';
+const PENUMBRA_NODE_PD_URL = 'https://grpc.testnet.penumbra.zone/';
```

Comment:
See my other comment, think we should import these

Reply:
now using dotenv to import